SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialComponentTest/SpatialComponentTest.h

```diff
@@ -46,5 +46,6 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialComponentTest : public ASpatialFunct
 	void CheckComponents(ASpatialComponentTestActor* Actor, int ExpectedServerId, int ExpectedClient1ComponentCount,
 						 int ExpectedClient2ComponentCount);
 	void CheckComponentsCrossServer(ASpatialComponentTestActor* Actor, int StartServerId, int EndServerId);
-	bool VerifyTestActorComponents(ASpatialComponentTestActor* Actor, int ExpectedComponentCount);
+	bool VerifyTestActorComponents(ASpatialComponentTestActor* Actor, int ExpectedComponentCount, const FString& Message);
+	int GetComponentsCount(ASpatialComponentTestActor* Actor);
```

Comment:
* Return int32?
* Should take const ref param rather than ptr?
* Function should be const?

Reply:
Even better - made it static.